elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -410,6 +414,7 @@ class PfeNavigation extends PFElement {
    * Clone them into the shadowRoot
    * @param {array} mutationList Provided by mutation observer
    */
+  // @todo: processLightDom seems to be firing twice, need to look into this and see whether that is okay or if it needs to be fixed, seems like it is not a good thing but not sure if it is avoidable or not
```

Comment:
Is it once per pfe-navigation on the page? Or 2x per pfe-navigation on the page?

Reply:
@wesruv I checked it with a `console.count()` and it seemed like it might be 2x per pfe-nav, however I think we need to double check that with only one pfe-nav on the page because I might have just misread the console output. Sorry! didn't think about checking the count with only one pfe-nav on the page when I was testing.